docs/src/index.rst

```diff
@@ -4,103 +4,40 @@
    contain the root `toctree` directive.
 .. highlight:: cmake
 
-ROCm CMake Build Tools
-======================
+.. meta::
+  :description: ROCm CMake
+  :keywords: ROCm, Cmake, library, api, AMD
 
-Summary
--------
+.. _rocm-cmake:
 
-ROCm CMake Build Tools (also stylized as "ROCmCMakeBuildTools" and previously
-known as "rocm-cmake") is a collection of functions that unify and simplify the
-CMake code of the ROCm components, as well as ensuring consistency across these
-components.
+****************************************************
+ROCm CMake build tools
+****************************************************
 
-Usage
------
+ROCm CMake build tools (also known as "rocm-cmake") is a collection of functions 
+that unify and simplify the CMake code of ROCm components, as well as ensuring 
+consistency across these different components. ROCm CMake build tools are primarily 
+used when building a library, and as such are not runtime dependencies for any generated 
```

Comment:
Instead of "primarily used when building a library," could we be more specific and mention that they're used during development too? e.g., "The ROCm CMake build tools are used when building and developing many AMD ROCm libraries, but are not runtime dependencies for any libraries, packages or executables."